packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md

```diff
@@ -728,64 +785,87 @@ class GroupedNotificationDrawer extends React.Component {
-              dropdownItems={userDropdownItems}
-            />
-          </PageHeaderToolsItem>
-        </PageHeaderToolsGroup>
-        <Avatar src={imgAvatar} alt="Avatar image" />
-      </PageHeaderTools>
+    const headerToolbar = (
+      <Toolbar>
+        <ToolbarContent>
+          <ToolbarGroup alignment={{ default: 'alignRight' }}>
+            <ToolbarGroup variant="icon-button-group">
+              <ToolbarItem visibility={{ default: 'visible' }} isSelected={isDrawerExpanded}>
+                <NotificationBadge
+                  variant={this.getNumberUnread() === 0 ? 'read' : 'unread'}
+                  onClick={this.onCloseNotificationDrawer}
+                  aria-label="Notifications"
+                >
+                  <BellIcon />
+                </NotificationBadge>
+              </ToolbarItem>
+              <ToolbarGroup
+                variant="icon-button-group"
+                visibility={{
+                  default: 'hidden',
+                  lg: 'visible'
+                }} /** the settings and help icon buttons are only visible on desktop sizes and replaced by a kebab dropdown for other sizes */
+              >
+                <ToolbarItem>
+                  <Button aria-label="Settings actions" variant={ButtonVariant.plain}>
+                    <CogIcon />
+                  </Button>
+                </ToolbarItem>
+                <ToolbarItem>
+                  <Button aria-label="Help actions" variant={ButtonVariant.plain}>
+                    <HelpIcon />
+                  </Button>
+                </ToolbarItem>
+              </ToolbarGroup>
+            </ToolbarGroup>
+            <ToolbarGroup>
+              <ToolbarItem
+                visibility={{
+                  lg: 'hidden'
+                }} /** this kebab dropdown replaces the icon buttons and is hidden for desktop sizes */
+              >
+                <Dropdown
+                  isPlain
+                  position="right"
+                  onSelect={this.onKebabDropdownSelect}
+                  toggle={<KebabToggle onToggle={this.onKebabDropdownToggle} />}
+                  isOpen={isKebabDropdownOpen}
+                  dropdownItems={kebabDropdownItems}
+                />
+              </ToolbarItem>
+              <ToolbarItem
+                visibility={{ default: 'hidden', md: 'visible' }} /** this user dropdown is hidden on mobile sizes */
+              >
+                <Dropdown
+                  isPlain
+                  position="right"
+                  onSelect={this.onDropdownSelect}
+                  isOpen={isDropdownOpen}
+                  toggle={<DropdownToggle onToggle={this.onDropdownToggle}>John Smith</DropdownToggle>}
+                  dropdownItems={userDropdownItems}
+                />
+              </ToolbarItem>
+            </ToolbarGroup>
+            <Avatar src={imgAvatar} alt="Avatar image" />
```

Comment:
Avatar should be part of the dropdown `https://www.patternfly.org/v4/components/dropdown#dropdown-with-image-and-text`